### ISSUE TITLE
Could we replace the links back to normal?

### ISSUE THREAD

**Issue by rolandschuetz** [networkteam/Networkteam.Neos.MailObfuscator#24]
In javascript we could add:

`requestAnimationFrame(function() {
 
 Array.prototype.forEach.call(document.querySelectorAll('a[href^="javascript:linkTo_UnCryptMailto"]'), function(link) { link.innerHTML = link.innerHTML.replace(' (at) ', '@')});
});`

**Comment by hlubek** [comment]
Good idea! Is there a reason why `requestAnimationFrame` should be used? I guess a dom ready event would be sufficient here.

We should put that behind an option though, I think there might be a number of crawlers with headless JS where that might disable the "protection"

**Comment by rolandschuetz** [comment]
I added requestAnimationFrame only to improve the speed, the transformation can happen after an initial paint to not slow things down.

I have no ideas about modern crawlers, for headless js probably " (at) " is also very obvious. But we could add a 3sec delay.
